src/main/java/org/folio/integration/http/VertxOkapiHttpClient.java

```diff
@@ -31,7 +36,8 @@ public CompletableFuture<Response> get(String path,
 
     final var request = client
       .get(url.getPort(), url.getHost(), url.getPath())
-      .putHeaders(buildHeaders(okapiHeaders));
+      .putHeaders(buildHeaders(okapiHeaders))
+      .timeout(5000);
```

Comment:
How does the rest of the system that uses this method handle the failure of the future?

Reply:
It seems to throw a 500 error with the text of the timeout error message.

Reply:
Thanks for checking.

IIRC it isn't easy to write a test that demonstrates this with the integrated code due to the custom fake server used for those tests.